pkg/minikube/config/profile.go

```diff
@@ -223,7 +223,25 @@ func ListProfiles(miniHome ...string) (validPs []*Profile, inValidPs []*Profile,
 	return validPs, inValidPs, nil
 }
 
-// removeDupes removes duplicates
+// ListProfilesLight returns profiles in minikube home dir
+// Unlike `ListProfiles` this function doens't try to get profile from container
+func ListProfilesLight(miniHome ...string) (ps []*Profile, err error) {
```

Comment:
I would rename this function to `ListValidProfiles` to more accurately describe what is happening

Reply:
Fixed.